wellregistry/registry/models.py

```diff
@@ -183,7 +181,7 @@ class MonitoringLocation(models.Model):
     alt_method = models.CharField(max_length=300, blank=True, verbose_name='Altitude method')
     alt_acy = models.CharField(max_length=300, blank=True, verbose_name='Altitude accuracy')
 
-    well_depth = models.DecimalField(max_digits=11, decimal_places=8, null=True, blank=True)
+    well_depth = models.DecimalField(max_digits=11, decimal_places=7, null=True)
```

Comment:
This field was not configure to hold values > 999 so I modified it. I looked in the legacy database and no well depths are larger than 9999.

Reply:
blank = True might need to be added back here? Well depth can be null in some situations.

Reply:
It's a decimal field so you don't need blank=True and null=True

Reply:
But now that I think about it, if we don't set blank=True then the field is required in the form. I will revert this change will doing the current ticket. Thanks for being so detail oriented on this!